build-system/tasks/firebase.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const colors = require('ansi-colors');
+const fs = require('fs-extra');
+const gulp = require('gulp-help')(require('gulp'));
+const path = require('path');
+
+
+function generateFirebaseFolder() {
+  fs.mkdirpSync('firebase');
+  if (argv.file) {
```

Comment:
would be cool to support globs and multiple files! (just a future feature request)

Reply:
Yeah this is currently my workflow for deploying anything for UX review and Viewer testing because firebase has a weird quirk with forcing index.html files. I think in the multiple files version we should just copy them into the `firebase folder`. Pending feedback, I think the most common usecase here is: build and deploy the minimum of everything needed for a `.amp.html` file.